### PR TITLE
revengenetwork

### PR DESCRIPTION
**Pull request by G4shetita**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/revengenetwork/background.png
Comment (maaaathis) [inline minecraft_servers/revengenetwork/background.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/revengenetwork/background.png
Comment (G4shetita) [reply minecraft_servers/revengenetwork/background.png:1]: compresed

File: minecraft_servers/revengenetwork/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "server_name": "revengenetwork",
+  "nice_name": "RevengeNetwork",
+  "direct_ip": "RevengeNetwork.es",
```
Comment (DoJapHD) [inline minecraft_servers/revengenetwork/manifest.json:4]: Lowercase IP

File: minecraft_servers/revengenetwork/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "server_wildcards": [
+    "%.RevengeNetwork.es"
```
Comment (DoJapHD) [inline minecraft_servers/revengenetwork/manifest.json:6]: Lowercase IP

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/revengenetwork/manifest.json (hunk 2 above)
Comment (G4shetita) [reply minecraft_servers/revengenetwork/manifest.json]: changed

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**